scripts/vep_backend/vep_110_conf.toml

```diff
@@ -0,0 +1,12 @@
+[workflow]
+sequencing_type = 'genome'
+use_vep_110 = true
+
+[vep]
+scatter_count = 1
+
+[references]
+vep_110_mount = 'gs://cpg-common-test/references/vep/110/mount'
```

Comment:
Should this be `VEP_110_mount` or simply `VEP_mount`? I know this one is a bit of a breaking change from the old deployment, but going forward I would be keen to end up in a situation where we have a single config which is `VEP_version` or similar where we can just choose from a list of supported versions (and cognate mounts). How do we move to that?

Regardless, can you please add any new config options into the defaults file.

Reply:
The main version of this path is set up in the references repo, so it's in the default config for all runs. I figured specifying the version in the key enables us to remain backwards compatible and add optional new versions.

This was the test mount specified while I was iterating on it, that can probably be deleted now..

`vep_110_mount = "gs://cpg-common-main/references/vep/110/mount"`